definitions/features/pulp2.rb

```diff
@@ -5,7 +5,9 @@ class Features::Pulp < ForemanMaintain::Feature
     label :pulp2
 
     confine do
-      find_package('pulp-server')
+      if check_max_version('katello', '4.0')
```

Comment:
wouldn't this be true for 4.0? 

```
      def check_max_version(name, maximal_version)
        check_version(name) do |current_version|
          version(maximal_version) >= current_version
        end
      end

```
should it be 3.18?

Reply:
Good point!